docs/guidelines/content-guidelines/02-style-and-terminology.md

```diff
@@ -45,24 +45,26 @@ Use the second person and the pronouns "you," "your," and "yours" to speak direc
 
 It's okay to use contractions in the documentation from time to time. However, do not overuse it. Otherwise, your text will become messy and hard to read.
 
-✅ It's okay to use contractions.
+✅ It's okay to use contractions.  
 ✅ It is okay to not use them as well.
 
 ### Articles
 
 Always verify the use of the articles "a", "an", and "the" where appropriate. Use "the" when you refer to a specific example. Use "a" when you refer to something non-specific or hypothetical.
 
-> **NOTE:** Use `Kyma` whenever you refer to the name of our product:   
-✅ Kyma is awesome.  
-⛔️ The Kyma is not awesome.
+> **NOTE:** Whenever you refer to the name of our product or one of our components, don't use an article:  
+  ✅ Kyma is awesome.  
+  ⛔️ The Kyma is not awesome.
+  ✅ Application Connector  
+  ⛔️ the Application Connector
 
 ### Punctuation
 
 Use colons and semicolons sparingly. Use the colon ( : ) to introduce a list of things. The semicolon ( ; ) separates two distinct clauses within one sentence. The phrase after a semicolon is a complete sentence. However, the preferred method is without a colon or semicolon.
```

Comment:
While mentioning colons, maybe we could also specify the preferred capitalization after the colon? I remember we discussed it once, whether we should capitalize what comes after the colon, or go with lowercase.

Reply:
I suggest following the [SAP style guide](https://help.sap.com/docs/SNG_FOR_UA/e33c591ae4494a659a3f5f983c9d1161/e2bfb395649b410ba05a213d80092de5.html?locale=en-US&state=DRAFT)?

Reply:
Honestly, I don't mind either. But I wonder if we should not stick to what we already discussed and have in the docs, for example [here](https://kyma-project.io/blog/2021/12/7/release-notes-20/)...